src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -25,7 +26,7 @@
      transform_boolean_for_table)
 
 # storage account commands
-factory = lambda kwargs: storage_client_factory().storage_accounts
+factory = lambda kwargs: storage_client_factory().storage_accounts  # noqa: E731 lambda vs def
```

Comment:
I let this go. However I prefer not to reuse the `factory` variable across code blocks.